GEECS-PythonAPI/geecs_python_api/controls/api_defs.py

```diff
@@ -32,29 +33,11 @@ class ScanTag(NamedTuple):
 
 def month_to_int(month: Union[str, int]) -> int:
     """ :return: an integer representing the given month """
-    month_map = {
-        'jan': 1, 'january': 1,
-        'feb': 2, 'february': 2,
-        'mar': 3, 'march': 3,
-        'apr': 4, 'april': 4,
-        'may': 5,
-        'jun': 6, 'june': 6,
-        'jul': 7, 'july': 7,
-        'aug': 8, 'august': 8,
-        'sep': 9, 'september': 9,
-        'oct': 10, 'october': 10,
-        'nov': 11, 'november': 11,
-        'dec': 12, 'december': 12,
-    }
     try:
         month_int = int(month)
         if 1 <= month_int <= 12:
             return month_int
```

Comment:
Do handle the case if month_int is outside 1..12, because otherwise dateparse(month) will return unexpected results. Consider checking isinstance(month, str) before the dateparse.

Reply:
Good call, added it in!